Assets/Scripts/UI/PickupPup/MainMenu/MainMenu.cs

```diff
@@ -179,11 +184,47 @@ public void OnGiftsClick()
 
     public void OnWatchAdClick()
     {
-        // Disabled
-        EventController.Event(k.GetPlayEvent(k.EMPTY));
+
+        #if UNITY_IOS
+                Advertisement.Initialize(IOS_ID, true);
```

Comment:
I decided to go with a regular if statment if(Advertisment.IsInitialized) instead so it doesnt matter if unity auto initializes or not.